package-lock.json

```diff
@@ -9673,6 +9673,7 @@
             "resolved": "https://registry.npmjs.org/bufferutil/-/bufferutil-4.0.5.tgz",
             "integrity": "sha512-HTm14iMQKK2FjFLRTM5lAVcyaUzOnqbPtesFIvREgXpJHdQm8bWS+GkQgIkfaBYRHuCnea7w8UVNfwiAQhlr9A==",
             "dev": true,
+            "hasInstallScript": true,
```

Comment:
nit: rebase as I don't think these changes are needed.

Reply:
I am not sure how this got marked as added.  Git says it was added by Nikolay 5 months ago.
<img width="976" alt="Screenshot 2023-07-11 at 9 28 20 PM" src="https://github.com/hashgraph/hedera-json-rpc-relay/assets/183383/926bcbe7-fdb1-4788-9407-fa87cea4593a">

Reply:
Strange. The `hasInstallScript` shouldn't be needed but the snapshot updates should already be in teh repo with [this PR ](https://github.com/hashgraph/hedera-json-rpc-relay/pull/1244/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519)